components/insight/SRC/org/openmicroscopy/shoola/agents/treeviewer/view/ToolBar.java

```diff
@@ -85,6 +84,7 @@
 import org.openmicroscopy.shoola.env.LookupNames;
 import org.openmicroscopy.shoola.env.data.model.ScriptObject;
 import org.openmicroscopy.shoola.env.ui.TaskBar;
+import org.openmicroscopy.shoola.util.CommonsLangUtils;
```

Comment:
Note this import will conflict with https://github.com/openmicroscopy/openmicroscopy/commit/7db2c2565e0b7094dc47f4febc83eb5280b46ffb#diff-1799b1647ef5bf4722c7f08b7b5c15e2R88 and the conflict will need to be resolved when merging `develop` into `regions` /cc @dominikl @mtbc